src/main/java/org/gridsuite/voltageinit/server/service/parameters/VoltageInitParametersService.java

```diff
@@ -174,7 +217,23 @@ public OpenReacParameters buildOpenReacParameters(VoltageInitRunContext context,
 
                 network.getVoltageLevelStream()
                     .filter(voltageLevel -> voltageLevelDefaultLimits.keySet().contains(voltageLevel.getId()) || voltageLevelModificationLimits.keySet().contains(voltageLevel.getId()))
-                    .forEach(voltageLevel -> fillSpecificVoltageLimits(specificVoltageLimits, voltageLevelModificationLimits, voltageLevelDefaultLimits, voltageLevel, context.getVoltageLevelsIdsRestricted()));
+                    .forEach(voltageLevel -> fillSpecificVoltageLimits(specificVoltageLimits,
+                        counterMissingVoltageLimits, counterVoltageLimitModifications,
+                        voltageLevelModificationLimits, voltageLevelDefaultLimits,
+                        voltageLevel, context.getVoltageLevelsIdsRestricted()));
+
+                if (!context.getVoltageLevelsIdsRestricted().isEmpty()) {
```

Comment:
It was in a method, why not keep it in a method?

Reply:
to keep all logs near where there calculated, not scattered in multiples files

Reply:
I agree, but we can keep it in a separate method in *this* class?

Reply:
oh, I see  
yes, we can, but did we gain a lot for 1 log line?

Reply:
like this?  
https://github.com/gridsuite/voltage-init-server/compare/add_logs...add_logs_tmp